R/arc-select.R

```diff
@@ -170,14 +211,17 @@ collect_layer <- function(x, n_max = Inf, token = Sys.getenv("ARCGIS_TOKEN"), ..
   suppressMessages(
     n_feats <- httr2::resp_body_json(
       httr2::req_perform(
-        httr2::req_url_query(n_req, f = "pjson")
+        httr2::req_url_query(n_req, f = "pjson"),
+        error_call = error_call
```

Comment:
error_call is not an argument of `req_perform()` nor does it accept dots. This looks liek a bug to me.

Reply:
It was made available in 0.2.2.9000. The most recent CRAN version is `0.2.3`. Its important to note that adding this dependency _forces_ a user to upgrade their version of httr2 to 0.2.3 which should be done with care. Since there are no breaking changes documented in the [NEWS.md](https://github.com/r-lib/httr2/blob/main/NEWS.md) I'm okay with this. You have to record the minimum version in the DESCRIPTION though.

Reply:
Totally. Meant to have that in with the original pull request but forgot last night.